docs/advanced-usage/compute-backends/kubernetes/index.md

```diff
@@ -17,3 +17,66 @@ on the default backend.
         commands:
             - python "${helloworld}"
 ```
+
+## Custom memory limit
+
+Kubernetes allows configuring and managing resources for containers. With REANA, you can configure cluster wide **memory** limits to make the effiecient use of available resources.
+
+To set the memory limit of a job you must specify `kubernetes_memory_limit`.
+
+This can be set directly in `bytes` and you can read more about the expected memory values on [kubernetes official documentation](https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/#meaning-of-memory)
+
+Note that jobs exceeding the memory limits, you will get out of memory (OOM) error and job gets [`OOMKilled`](https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/#requests-and-limits).
+
+```yaml hl_lines="7"
```

Comment:
Please add some text to introduce the examples, and mention which file it is in each case.